autometa/taxonomy/ncbi.py

```diff
@@ -392,13 +404,16 @@ def is_valid_taxid(self, taxid):
         -------
         int
             `taxid` if the `taxid` is a positive integer and present in either nodes.dmp or names.dmp or
-            new_taxid which is equal to the secondary `taxid` present in merged.dmp
+            merged `taxid`
```

Comment:
You could be more descriptive here rather than merged `taxid`.

i.e. taxid recovered from merged.dmp